cranelift-codegen/meta/src/cdsl/operands.rs

```diff
@@ -116,7 +116,7 @@ pub(crate) enum OperandKindFields {
 pub(crate) struct OperandKind {
     pub name: &'static str,
     doc: Option<&'static str>,
-    default_member: Option<&'static str>,
+    format_field_name: Option<&'static str>,
```

Comment:
I'm not sure I understand why this is called `format_field_name` and not just `field_name`--is it because it is related to the format somehow? We might need to document why this struct field is needed and what it is used for.

Reply:
Or `rust_field_name`...